src/main/java/it/eng/idsa/businesslogic/processor/sender/registration/SenderCreateQueryBrokerMessageProcessor.java

```diff
@@ -26,24 +26,12 @@ public void process(Exchange exchange) throws Exception {
 		Map<String, Object> receivedDataHeader = exchange.getMessage().getHeaders();
```

Comment:
Can we remove this receivedHeaders, since we do not do anything special with headers?
What happens if we pass all incoming headers to broker?